sbin/Makefile.am

```diff
@@ -9,20 +7,19 @@ scriptsin = \
 	update-ipsets.in \
 	vnetbuild.in
 
-inclibdir = $(libarchinddir)/firehol
+inclibdir = $(libexecdir)/firehol/@PACKAGE_VERSION@
```

Comment:
I'm not sure I understand. Do you mean allow specifying whole directory via ./configure or something else?